### ISSUE TITLE
[Bug]: Once set up biometric, app crashes

### ISSUE THREAD

**Issue by JeanlChristophe** [synonymdev/bitkit-react-native#1087]
### Describe the bug

Once set up biometric, the screen is all black, nothing works anymore. Even after forcing close the application 10 times, I still have a black screen.

It already happened to a user earlier : https://app.asana.com/0/1203624355958644/1204377573367004/f

### Reproduce

1. Go to 'Settings'
2. Click on 'Security and 'privacy'
3. Click on 'Use biometrics instead'

### Operating system

Graphene OS

### Bitkit version

v49

**Comment by pwltr** [comment]
@JeanlChristophe Can you please post a recording of this bug? Do you have a fingerprint setup on your phone?

**Comment by JeanlChristophe** [comment]
I no longer have access to the profile with which I had the bug.
And with a new profile, I can't reproduce the error. It happens from time to time, but rarely.

**Comment by JeanlChristophe** [comment]
From support, we have a user having a issue:

"I have activated biometrics authentication, but it crashes the app every time I log in.
Recently, it started working again (no crash after logging in) but that was only temporarily, and the problem started again.

Restoring on another device did not work. It's stuck here. See attached sorry clip"



https://github.com/synonymdev/bitkit/assets/46056211/1afbca98-534f-40d1-850e-5bc9dc55f9bb

**Comment by pwltr** [comment]
@JeanlChristophe 

I need more information here as I cannot reproduce this issue on my phone and am flying completely blind.

- When does the crash occur? Immediately after enabling biometrics? Only when starting the app?
- Which biometric method are you trying to setup? Fingerprint or face recognition?
- Does the screen just stay black or does the app crash? Is there an error screen/message?
- Please provide a screen recording of the issue

Please answer above questions thoroughly, with any luck this will be fixed with https://github.com/synonymdev/bitkit/pull/1110

**Comment by JeanlChristophe** [comment]
JC:
-     When does the crash occur? Immediately after enabling biometrics
-     Which biometric method are you trying to setup? Fingerprint
-     The screen just stay black without an error screen/message

Support user:
-     When does the crash occur? Immediately after enabling biometrics
-     Which biometric method are you trying to setup? Fingerprint
-     The app crash after 5 sec max without an error screen/message

**Comment by pwltr** [comment]
Let's retest this with the next release to see if https://github.com/synonymdev/bitkit/pull/1114 fixed it.

**Comment by thisispav** [comment]
tried to reproduce this bug, it's not working so far. I am using **Pixel 6A (running on Graphene).**

I then went to:
Go to 'Settings'
Click on 'Security and 'privacy'
Click on 'Use biometrics instead'

but I don't even see the option to enable biometrics / Use biometrics instead.

I uninstalled / installed the app several times to make sure it's not something related to the device. I then removed the APK and re-installed it again, to no avail.

https://github.com/synonymdev/bitkit/assets/119061223/8b8380af-0e02-4e83-b73a-b0cf93d65376

With **iPhone 6S Pro - seems to be working fine**

**Comment by thisispav** [comment]
**tested with iPhone 6S Pro** - after app installation, I enabled Biometrics from Settings > Security - works fine
I then disabled the option - works as intended.
Enabled the option again, without problem.

No crashes, black screens or unintended behavior. 

**tested with Pixel 6A (running on Graphene)** - after app installation, I enabled Biometrics from Settings > Security - works fine
re-tested the same scenario another 3 times. Uninstalled and installed the app and followed the same flow. Uninstalled and reinstalled the APK to try and reproduce the bug
